jest-setup.js

```diff
@@ -1,136 +1,162 @@
-const _ = require("lodash");
-const basicChecks = require("stylelint").basicChecks;
-const less = require("postcss-less");
-const stylelint = require("stylelint").standalone;
+'use strict';
 
+const basicChecks = require('stylelint').basicChecks;
+const stylelint = require('stylelint').lint;
+const util = require('util');
 
-jest.mock("./getOsEol.js", () => () => "\n");
+jest.mock('./getOsEol', () => () => '\n');
 
 global.testRule = (rule, schema) => {
```

Comment:
BTW, why do we have `rule` here? It's not used anywhere. It looks like it was added long time ago, and since then never had been removed.

Reply:
I don't know. I'll remove it in this pull request.